force-app/main/default/lwc/donationHistoryFilter/donationHistoryFilter.js

```diff
@@ -18,7 +19,12 @@ export default class DonationHistoryFilter extends LightningElement {
     value = LIFETIME_VALUE;
     
     connectedCallback() {
-        getYearsWithDonation({contactId : this.contactId})
+        this.getYearsByContact();
+    }
+
+    getYearsByContact(){
```

Comment:
Why did you change it? I dont see the point and also the name getYearsByContact is more confusing than getYearsWithDonations. If you wanted to rename it, it would be better getYearsWithDonationsByContact